Src/BootCamp.Chapter/ShuffleDemo.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace BootCamp.Chapter
+{
+    internal static class ShuffleDemo
+    {
+        internal static void Demo()
```

Comment:
Remove it- not a part of this class. It's a part of other class where you demo it. You couple too much here.

Reply:
oke, so if I have put the extension in the right class , I still have to delete that one.
or do you want one demo class where I call all the three/four demo's

Reply:
Extension has nothing to do with demo.

You always need at least 2 classes: 
One for demo
One for implementation 

By implementing it in the same class as demo you couple demo with the thing it demos.

Reply:
oke, will do that